.azure/pipelines/jobs/default-build.yml

```
@@ -139,6 +139,11 @@ jobs:
         - _SignType: real
       - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
         - _SignType: test
+    - ${{ if eq(parameters.agentOs, 'Linux') }}:
+      - ${{ if eq(parameters.useHostedUbuntu, true) }}:
+        - LC_ALL: 'en_US.UTF-8'
+        - LANG: 'en_US.UTF-8'
+        - LANGUAGE: 'en_US.UTF-8'
```

nit: It doesn't hurt anything to define these environment variables unconditionally. If it were necessary, my nit would have been to use `and(...)` rather than two conditions. As it is,
```suggestion
    - LC_ALL: 'en_US.UTF-8'
    - LANG: 'en_US.UTF-8'
    - LANGUAGE: 'en_US.UTF-8'
```